File: apps/boardest/replace_io.py
import os
import glob

def replace_dart_io():
    base_dir = os.path.abspath('lib')
    dart_files = glob.glob(os.path.join(base_dir, '**', '*.dart'), recursive=True)
    
    count = 0
    for file_path in dart_files:
        with open(file_path, 'r', encoding='utf-8') as f:
            content = f.read()
            
        if "import 'dart:io';" in content:
            new_content = content.replace("import 'dart:io';", "import 'package:universal_io/io.dart';")
            with open(file_path, 'w', encoding='utf-8') as f:
                f.write(new_content)
            count += 1
            print(f"Replaced in {file_path}")
            
    print(f"Total {count} files updated.")

if __name__ == '__main__':
    replace_dart_io()

File: apps/boardest/replace_io.dart
import 'dart:io';

void main() {
  final libDir = Directory('lib');
  if (!libDir.existsSync()) {
    print('lib directory not found');
    return;
  }

  int count = 0;
  final files = libDir.listSync(recursive: true);
  for (final file in files) {
    if (file is File && file.path.endsWith('.dart')) {
      final content = file.readAsStringSync();
      if (content.contains("import 'dart:io';")) {
        final newContent = content.replaceAll("import 'dart:io';", "import 'package:universal_io/io.dart';");
        file.writeAsStringSync(newContent);
        count++;
        print('Replaced in ${file.path}');
      }
    }
  }
  print('Total $count files updated.');
}
